src/pages/projects/ProjectsFilterAndSort.tsx

```diff
@@ -21,6 +22,8 @@ export default function ProjectsFilterAndSort({
   setIncludeV2,
   showArchived,
   setShowArchived,
+  searchTags,
```

Comment:
adding tags to project sort/filter options. this adds a new tags dropdown selector, allowing the user to check which tags to filter by, if any